README.md

```diff
@@ -1,444 +1,89 @@
-
-<hr>
-
-## Overview
-
-### Systems and the Executor-Trainer Paradigm
-
-At the core of the Mava framework is the concept of a `system`. A system refers to a full multi-agent reinforcement learning algorithm consisting of the following specific components: an `Executor`, a `Trainer` and a `Dataset`.
+<p align="center">
+    <a href="docs/images/mava.png">
+        <img src="docs/images/mava.png" alt="Mava logo" width="70%"/>
+    </a>
+</p>
 
-The `Executor` is the part of the system that interacts with the environment, takes actions for each agent and observes the next state as a collection of observations, one for each agent in the system. Essentially, executors are the multi-agent version of a typical Actor class in single-agent RL and are themselves constructed through feeding to the executor a dictionary of policy networks. The `Trainer` is responsible for sampling data from the Dataset originally collected from the executor and updating the parameters for every agent in the system. Trainers are therefore the multi-agent version of the typical Learner class. The `Dataset` stores all of the information collected by the executors in the form of a collection of dictionaries for the actions, observations and rewards with keys corresponding to the individual agent ids. The basic system design is shown on the left in the above figure.
-Several examples of system implementations can be viewed [here][Systems].
+<h2 align="center">
+    <p>A research framework for distributed multi-agent reinforcement learning in JAX</p>
+</h2>
+
+<div align="center">
+<a rel="nofollow">
+    <img src="https://img.shields.io/pypi/pyversions/id-mava" alt="Python" />
+</a>
+<a rel="nofollow">
+    <img src="https://badge.fury.io/py/id-mava.svg" alt="PyPi" />
+</a>
+<a rel="nofollow">
+    <img src="https://github.com/instadeepai/Mava/workflows/format_and_test/badge.svg" alt="Formatting" />
+</a>
+<a rel="nofollow">
+    <img src="https://img.shields.io/lgtm/grade/python/g/instadeepai/Mava.svg?logo=lgtm&logoWidth=18" alt="Quality" />
+</a>
+<a rel="nofollow">
+    <img src="https://img.shields.io/badge/License-Apache%202.0-blue.svg" alt="License" />
+</a>
+<a rel="nofollow">
+    <img src="https://readthedocs.org/projects/id-mava/badge/?version=latest" alt="Docs" />
+</a>
+</div>
 
```

Comment:
a little strange having this graphic immediately after the heading without any explanation (though it is a beautiful one)

Reply:
I think the point is more to grab attention and a full explanation can be found in the docs. But we could also add a small bit about the distributed setup in the readme if you think that would be better?

Reply:
We could add text but I guess this is fine as we link to the docs and the graphic is explained on the home page